redmine/src/main/java/org/arquillian/extension/governor/redmine/impl/reporter/RedmineGovernorRecorder.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2016, Red Hat, Inc. and/or its affiliates, and individual
+ * contributors by the @authors tag. See the copyright.txt in the
+ * distribution for a full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.arquillian.extension.governor.redmine.impl.reporter;
+
+import org.arquillian.extension.governor.redmine.api.Redmine;
+import org.arquillian.extension.governor.redmine.configuration.RedmineGovernorConfiguration;
+import org.arquillian.recorder.reporter.event.PropertyReportEvent;
+import org.arquillian.recorder.reporter.model.entry.KeyValueEntry;
+import org.arquillian.recorder.reporter.model.entry.table.TableCellEntry;
+import org.arquillian.recorder.reporter.model.entry.table.TableEntry;
+import org.arquillian.recorder.reporter.model.entry.table.TableRowEntry;
+import org.jboss.arquillian.core.api.Event;
+import org.jboss.arquillian.core.api.Instance;
+import org.jboss.arquillian.core.api.annotation.Inject;
+import org.jboss.arquillian.core.api.annotation.Observes;
+import org.jboss.arquillian.test.spi.TestClass;
+import org.jboss.arquillian.test.spi.event.suite.After;
+
+import java.lang.reflect.Method;
+
+/**
+ * @author <a href="mailto:dpawar@redhat.com">Dipak Pawar</a>
```

Comment:
I'm not sure if we should have authors in the code, as it's collective code ownership afterwards. But I guess it won't harm to add Apache license headers like in the [Arquillian Core](https://github.com/bartoszmajsak/arquillian-core/blob/b47c8779fe2e56e0714bf840e178d4d146e2da88/junit/core/src/test/java/org/jboss/arquillian/junit/rules/AbstractRuleStatementEnrichment.java#L1-L16). WDYT @lordofthejars @smiklosovic ?